### PR TITLE
RT-5.1 add Ia deviation IPv4(6)/Neighbor

### PR DESCRIPTION
**Pull request by mojiiba**
this pull adds a IPNeighborMissing to skip IPv4(6)/Neighbor validation.

### CODE REVIEW

**Review by thesrinath — COMMENTED** [review]
/gcbrun

**Review by thesrinath — APPROVED** [review]

File: feature/interface/singleton/ate_tests/singleton_test/singleton_test.go [hunk 1]
```diff
@@ -219,16 +219,20 @@ func (tc *testCase) verifyInterfaceDUT(
...
-	// IPv4 neighbor discovered by ARP.
-	dis4np := disp.Ipv4().Neighbor(atea.IPv4)
-	if got := gnmi.Get(t, tc.dut, dis4np.Origin().State()); got != dynamic {
-		t.Errorf("%s IPv4 neighbor %s origin got %v, want %v", dp, atea.IPv4, got, dynamic)
-	}
+	if !*deviations.IPNeighborMissing {
+		// IPv4 neighbor discovered by ARP.
+		dis4np := disp.Ipv4().Neighbor(atea.IPv4)
+		neigbour := gnmi.Get(t, tc.dut, dis4np.State())
+		fmt.Printf("%v", neigbour)
```
Comment (greg-dennis) [inline feature/interface/singleton/ate_tests/singleton_test/singleton_test.go:226]: I think this was a stray debugging line that was meant to be deleted. If it was meant to be kept, it should use t.Log. Using fmt.Printf is actually breaking Google's internal Go test log parsing :)

Comment (greg-dennis) [reply feature/interface/singleton/ate_tests/singleton_test/singleton_test.go:226]: I created https://github.com/openconfig/featureprofiles/pull/1007 to remove the debugging lines that I think were unintentionally left in

**Pull request merged**